packages/back-end/src/index.ts

```diff
@@ -1,7 +1,13 @@
 import express from "express";
+import DatabaseServiceImpl from "./entities/database";
+import neo4j from "neo4j-driver";
 
 export default function createApp() {
   const app = express();
+  app.set(
+    'databaseService',
+    new DatabaseServiceImpl(neo4j.driver('bolt://localhost:7687'))
+  );
```

Comment:
I would have gone with
```suggestion
export default function createApp(databaseService: DatabaseService) {
  const app = express();
  app.set(
    'databaseService',
    databaseService
  );
```
In order to keep the dependency independant from the actual app, so that upon creating the app you can send the implemented dependency or a mock